docs/reference/content/bson/extended-json.md

```diff
@@ -116,5 +116,29 @@ try {
 The `Document.parse()` static factory method constructs an instance of a `JsonReader` with the given string and returns an instance of an
 equivalent Document. `JsonReader` automatically detects the JSON flavor in the string, so you do not need to specify it.
 
+### Reading Directly to JSON
```

Comment:
The example shows both writing JSON directly to MongoDB and reading JSON directly from MongoDB.  Let's clarify in the text above the example that both are supported.

Reply:
Done